ide-common/src/main/java/org/digma/intellij/plugin/log/Log.java

```diff
@@ -23,6 +23,15 @@ public static void log(Consumer<String> consumer, String format, Object... args)
         consumer.accept(DIGMA + String.format(format.replace("{}", "%s"), args));
     }
 
+    public static void test(Consumer<String> consumer, String format, Object... args) {
+        StackTraceElement[] stackTraceElements = Thread.currentThread().getStackTrace();
+        String callingMethodName = stackTraceElements[2].getMethodName();
+        String lineNumber = String.valueOf(stackTraceElements[2].getLineNumber());
+        String threadName = Thread.currentThread().getName();
+        String header = String.format("%s:%s - %s", callingMethodName, lineNumber, threadName);
+        consumer.accept(header + " - " + String.format(format.replace("{}", "%s"), args));
+    }
+
     public static void debugWithException(Logger logger,Throwable e, String format, Object... args) {
```

Comment:
this log method is called to much from all over and will explode the user log.
getting stack trace is a heavy operation and should be called only if log level is enabled. if called with info it will explode the user log.